test/quantization/fx/test_model_report_fx.py

```diff
@@ -1093,55 +1093,63 @@ def test_generate_report(self):
 
 class TestFxDetectInputWeightEqualization(QuantizationTestCase):
 
-    @skipIfNoFBGEMM
-    def test_input_weight_equalization_determine_points(self):
-        class LinearConv(torch.nn.Module):
-            def __init__(self):
-                super().__init__()
-                self.linear = torch.nn.Linear(3, 3)
-                self.relu = torch.nn.ReLU()
-                self.conv2d = torch.nn.Conv2d(3, 3, 1)
+    class LinearConv(torch.nn.Module):
```

Comment:
potential nit: if you use similar things for many test cases, you can just have those defined outside the test class